server/shared/get_user_info_dict.py

```diff
@@ -0,0 +1,13 @@
+from social_django.models import UserSocialAuth
```

Comment:
A `shared/` directory isn't really idiomatic django. Can we just leave it in server.views? It seems more like a view-level helper function to me.